pyproject.toml

```
@@ -44,7 +45,7 @@ exclude = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
```

```suggestion
python = "^3.9"
```
and remove `"Programming Language :: Python :: 3.8",`